optuna/visualization/matplotlib/_pareto_front.py

```diff
@@ -74,13 +81,55 @@ def objective(trial):
     """
 
     _imports.check()
-
-    if len(study.directions) == 2:
-        return _get_pareto_front_2d(study, target_names, include_dominated_trials, axis_order)
-    elif len(study.directions) == 3:
-        return _get_pareto_front_3d(study, target_names, include_dominated_trials, axis_order)
+    if axis_order:
+        warnings.warn(
+            "``axis_order`` option is deprecated. This feature will be removed in the future.",
```

Comment:
A nit, but warning messages in console are not highlighted by the double back-ticks. Could you replace it with a single backtick?
```suggestion
            "`axis_order` option is deprecated. This feature will be removed in the future.",
```

Reply:
Thanks for your update!